<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>905</width>
    <height>572</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>905</width>
    <height>572</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>905</width>
    <height>572</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MDMA - Motion Detection Midi Interface - Unamed configuration</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../ressources.qrc">
    <normaloff>:/icons/ressources/Logo-256.png</normaloff>:/icons/ressources/Logo-256.png</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="central_widget">
   <widget class="QPushButton" name="pushButton_run">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>5</y>
      <width>250</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QTreeWidget" name="treeWidget_list">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>80</y>
      <width>250</width>
      <height>360</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Type</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Tab</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pushButton_deleteAll">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>498</y>
      <width>250</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete All</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_tab">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>645</width>
      <height>35</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Tab 1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Tab 2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Tab 3</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_delete">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>471</y>
      <width>250</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_configure">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>40</y>
      <width>250</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>Configure</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_edit">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>444</y>
      <width>250</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_camera">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>45</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_zone">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>45</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>905</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFicher">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_MDMA"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuFicher"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAbout_MDMA">
   <property name="text">
    <string>About MDMA</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>comboBox_tab</tabstop>
  <tabstop>pushButton_run</tabstop>
  <tabstop>pushButton_configure</tabstop>
  <tabstop>treeWidget_list</tabstop>
  <tabstop>pushButton_edit</tabstop>
  <tabstop>pushButton_delete</tabstop>
  <tabstop>pushButton_deleteAll</tabstop>
 </tabstops>
 <resources>
  <include location="../ressources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
